### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by benfletcher**
Typo fixes and minor re-wording.

### CODE REVIEW

File: www/src/pages/getting-started/theming.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ export default withLayout(function ThemingSection() {
       <p>
         Generally, if you stick to the bootstrap defined classes and variants,
         there isn't anything you need to do to use a custom theme with
-        ReactBootstrap, it just works. There are plently of cases tho where you
-        want to color outside the lines and we try to make that easy to do.
+        ReactBootstrap. It just works. But we also make coloring outside the
```
Comment (taion) [inline www/src/pages/getting-started/theming.js]:
```suggestion
        React-Bootstrap. It just works. But we also make coloring outside the
```

File: www/src/pages/getting-started/theming.js [hunk 2]
```diff
@@ -17,8 +17,8 @@ export default withLayout(function ThemingSection() {
       <p>
         Generally, if you stick to the bootstrap defined classes and variants,
```
Comment (taion) [inline www/src/pages/getting-started/theming.js]:
```suggestion
        Generally, if you stick to the Bootstrap-defined classes and variants,
```

**Comment by mxschmitt** [comment]
@benfletcher please apply the suggestions, without that we can't merge.

**Comment by taion** [comment]
or turn on maintainer edits

**Pull request closed without merging**